packages/gatsby-admin/src/prism-theme.tsx

```diff
@@ -0,0 +1,189 @@
+export default `
+/*
```

Comment:
This is used imported `pages/plugins.tsx`! I opted to use emotion's `<Global />` component to inject this CSS in order to avoid two ways to use CSS in Admin, as well as the additional tooling/configuration overhead. 👍